src/serialize.js

```diff
@@ -1,41 +1,22 @@
-function encode(data) {
-  return Buffer.from(prepareUnicodeEncode(JSON.stringify(data))).toString(
-    'base64'
-  )
-}
-module.exports.encode = encode
+const utf8 = require('utf8')
+const Buffer = require('buffer/').Buffer
```

Comment:
typo here, no idea how tests would be passing

Reply:
I'm guessing you're referring to the `/` in the require? this forces node to look inside `node_modules` instead of using `buffer` from the internal api